memor/prompt.py

```diff
@@ -62,14 +62,14 @@ def __init__(
         self._template = PresetPromptTemplate.DEFAULT.value
         self._responses = []
         self._selected_response_index = 0
-        if file_path:
+        if file_path is not None:
             self.load(file_path)
         else:
             if message:
                 self.update_message(message)
             if role:
                 self.update_role(role)
-            if tokens:
+            if tokens is not None:
                 self.update_tokens(tokens)
             if responses:
                 self.update_responses(responses)
```

Comment:
if someone updates the responses with `[]` that's also the same thing. So we should update this as well.

Reply:
fixed in https://github.com/openscilab/memor/pull/146/commits/1b38cc83fe1af630ba7ed48c42b7485da88b6196 and https://github.com/openscilab/memor/pull/146/commits/663fe6207ceec08d21e8da709286cf0ea1a38f6a

Reply:
Since `""` and `[]` evaluate to **False** in Python, I applied our approach for both cases.